templates/project/CONTRIBUTING.md

```diff
@@ -93,13 +93,18 @@ php-cs-fixer fix --verbose
 Please note that we try to keep phpdoc to a minimum, so if an `@param` phpdoc
 comment brings nothing more than the type hint and variable name already do,
 it SHOULD be removed. Descriptions are OPTIONAL if you want to document a type.
+If you want to use pseudo-types, generics or templates for static-analysis,
+prefer the usage of `@phpstan-` annotations over classic annotations to keep
+IDE support, and over `@psalm-` annotations for consistency.
```

Comment:
We should keep a note, that you should keep the classic type hints and add additional type hints / generics via phpstan or psalm.

Reply:
Do you have any proposal about the text of the note ? I already don't really like the big sentence I made.